app/src/main/res/values/strings.xml

```diff
@@ -46,34 +46,34 @@
     <string name="d_add_fail_title">Fail</string>
     <string name="d_add_fail_text">Couldn\'t find a URL in share string:\n</string>
     <string name="d_configurationChanged_title">Parameters changed</string>
-    <string name="d_configurationChanged_message">It seems some parameters have been changed. It is recommended to perform a connection test.</string>
+    <string name="d_configurationChanged_message">Test the connection to pick up new changes.</string>
     <string name="d_configurationChanged_answer_decline">Not now</string>
     <string name="d_configurationIsQuestionable_title">Problem detected</string>
-    <string name="d_configurationIsQuestionable_message">There was a connection error recently. It is recommended to perform a connection test.</string>
+    <string name="d_configurationIsQuestionable_message">Test the connection to see if the recent connection error was a fluke.</string>
     <string name="addLink_success_text">Added</string>
-    <string name="settings_parametersAreOk">Parameters are fine</string>
+    <string name="settings_parametersAreOk">The parameters are fine</string>
     <string name="settings_parametersAutofilled">Parameters filled</string>
     <string name="testConnection_progressDialogMessage">Testing connection…</string>
     <string name="testConnection_errorMessage_incorrectUrl">Incorrect URL</string>
-    <string name="testConnection_errorMessage_unsupportedServerVersion">Unsupported server version detected</string>
-    <string name="testConnection_errorMessage_wallabagNotFound">wallabag service not found — check URL</string>
-    <string name="testConnection_errorMessage_httpAuth">HTTP Authentication failed</string>
-    <string name="testConnection_errorMessage_noCSRF">Error parsing login form</string>
+    <string name="testConnection_errorMessage_unsupportedServerVersion">The server version is unsupported</string>
+    <string name="testConnection_errorMessage_wallabagNotFound">Could not find wallabag service — check URL</string>
+    <string name="testConnection_errorMessage_httpAuth">Could not authenticate via HTTP(S)</string>
+    <string name="testConnection_errorMessage_noCSRF">Could not parse login form</string>
     <string name="testConnection_errorMessage_incorrectCredentials">Can\'t log in — check username and password</string>
-    <string name="testConnection_errorMessage_authProblems">Authorization-related problems. These usually caused by HTTP-redirects.</string>
-    <string name="testConnection_errorMessage_unknownPageAfterLogin">Unknown page after login. This is usually caused by enabled two-factor authentication, which is not yet supported by the app</string>
+    <string name="testConnection_errorMessage_authProblems">Could not log in. Make sure there are no HTTP-redirects.</string>
+    <string name="testConnection_errorMessage_unknownPageAfterLogin">Unknown page after login. Usually caused by two-factor authentication, (which the app doesn\'t support yet).</string>
```

Comment:
Supported now?

Reply:
[Nope](https://github.com/wallabag/android-app/issues/359).

Reply:
This issue is resolved with answering the question, i suppose